roles/download/tasks/download_container.yml

```diff
@@ -10,32 +10,25 @@
   tags:
     - facts
 
-# FIXME(mattymo): In Ansible 2.4 omitting download delegate is broken. Move back
-#                 to one task in the future.
+- debug:
```

Comment:
If you want to leave debug task in code, add verbosity, so it executes only when you provide - v

Reply:
agreed & fixed - removed unneeded debug message